ajax/libs/nipplejs/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "nipplejs",
+  "version": "0.6.7",
+  "filename": "nipplejs.min.js",
+  "description": "A virtual joystick for touch capable interfaces",
+  "author": "Yoann Moinet <yoann.moinet@gmail.com>",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/yoannmoinet/nipplejs.git"
+  },
+  "homepage": "http://yoannmoinet.github.io/nipplejs",
+  "keywords": [
+    "virtual",
+    "joystick",
+    "touch"
+  ],
+  "npmName": "nipplejs",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
+      ]
+    }
+  ]
```

Comment:
* You should be using git auto-update here since the [GitHub repo](https://github.com/yoannmoinet/nipplejs) has one release more than the [npm package](https://www.npmjs.com/package/nipplejs/).
* We don't need the `.eslintrc` file in `dist`, so exclude it in the [glob](https://github.com/isaacs/node-glob#glob-primer).

```
"autoupdate": {
    "source": "git",
    "target": "https://github.com/yoannmoinet/nipplejs.git",
    "fileMap": [
      {
        "basePath": "dist",
        "files": [
          "**/*.!(eslintrc)"
        ]
      }
    ]
  }
```